src/server/routes/sessions.js

```diff
@@ -47,6 +48,12 @@ router.get('/logout', function (req, res) {
 })
 
 router.post('/', async function (req, res, next) {
+  if (LOGIN_DISABLED_MESSAGE) {
+    res.json({
+      success: false,
+      message: `Login disabled: ${LOGIN_DISABLED_MESSAGE}`
+    })
```

Comment:
```suggestion
    return res.json({
      success: false,
      message: `Login disabled: ${LOGIN_DISABLED_MESSAGE}`
    })
```

I assume you mean to actually prevent sending the email in this case?  In which case, this line should return.